### PR TITLE
add flux operator python sdk example

### PR DESCRIPTION
**Pull request by vsoch**
This will add a Python example for using Kueue with the Flux Operator! I have most of the MPI Operator done too, but ran into a bug with the ssh secret volume.

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Users likely want to integrate Kueue into Python-based workflow tools, and examples are helpful for that!

#### Which issue(s) this PR fixes:

Issue has been closed, previous issue is:
Fixes https://github.com/kubernetes-sigs/kueue/issues/947

#### Special notes for your reviewer:

This worked so well! I'm excited to try this out in an actual workflow tool

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by kannon92 — COMMENTED** [review]
This looks good! But one thing I wonder is of should we have the same script duplicated in two places?  Can you either have the task readme just run the python file or use the inline definition in the readme

**Comment by vsoch** [comment]
> This looks good! But one thing I wonder is of should we have the same script duplicated in two places? Can you either have the task readme just run the python file or use the inline definition in the readme

Yeah - it was the same for the first example added (already duplicated). I think they are important in both places, so my plan is to get all three in (in this manner) and then update the site to have some javascript that will retrieve the file dynamically. It's how the examples / files are rendered here: https://converged-computing.github.io/flux-go/#pancakes-examples so hopefully I can figure out the same with hugo (I assume I can put html in markdown?)

**Comment by alculquicondor** [comment]
let's merge #969 first

**Comment by vsoch** [comment]
> let's merge https://github.com/kubernetes-sigs/kueue/pull/969 first

Nice!! Thank you @moficodes for adding that, I totally love it.

**Comment by moficodes** [comment]
@vsoch The include shortcode PR got merged. Could you rebase your PR and use include shortcode for the samples.

**Comment by vsoch** [comment]
okay we are rebased!

File: site/content/en/docs/tasks/run_python_jobs.md [hunk 1]
```diff
@@ -159,3 +159,120 @@ print(job)
...
+```
+```console
+📦️ Container image selected is ghcr.io/flux-framework/flux-restful-api...
+⭐️ Creating sample job with prefix hello-world...
+Use:
+"kubectl get queue" to see queue assignment
+"kubectl get pods" to see pods
+```
+
+You'll be able to almost immediately see the MiniCluster job admitted to the queue:
```
Comment (trasc) [inline site/content/en/docs/tasks/run_python_jobs.md:195]: nit

```suggestion
You'll be able to almost immediately see the MiniCluster job admitted to the local queue:
```

File: site/content/en/docs/tasks/run_python_jobs.md [hunk 2]
```diff
@@ -159,3 +159,120 @@ print(job)
 See the [BatchV1](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/BatchV1Api.md)
 API documentation for more calls.
 
+
+### Flux Operator Job
+
```
Comment (trasc) [inline site/content/en/docs/tasks/run_python_jobs.md:164]: nit:

can you also add some refereces to and from [Run A Flux MiniCluster](/docs/tasks/run_flux_minicluster/)

**Review by alculquicondor — COMMENTED** [review]
/approve

File: site/static/examples/python/sample-flux-operator-job.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            generate_name=job_name,
+            namespace="default",
+        ),
+        spec=spec,
+    )
+
+
+def main():
+    """
+    Run an MPI job. This requires the MPI Operator to be installed.
```
Comment (alculquicondor) [inline site/static/examples/python/sample-flux-operator-job.py:91]: Is this accurate? Shouldn't it say flux operator?

Comment (vsoch) [reply site/static/examples/python/sample-flux-operator-job.py:91]: yep that's a typo. I'll do a quick PR to fix.

Comment (vsoch) [reply site/static/examples/python/sample-flux-operator-job.py:91]: Wooo I got PR number 1000!! https://github.com/kubernetes-sigs/kueue/pull/1000 Do I get a special prize? :cookie:

Comment (alculquicondor) [reply site/static/examples/python/sample-flux-operator-job.py:91]: hahaha nice! Uhm.... Here's a 🥇

Comment (vsoch) [reply site/static/examples/python/sample-flux-operator-job.py:91]: Can I eat that? :laughing: 

Just kidding :) Thanks for the quick review! And overall for everything you do for these projects. <3

**Pull request merged**